code/modules/tooltip/tooltip.dm

```diff
@@ -81,15 +81,16 @@ Notes:
 
 /datum/tooltip/proc/hide()
 	if (src.queueHide)
-		spawn(1)
-			winshow(src.owner, src.control, 0)
+		addtimer(CALLBACK(src, .proc/do_hide), 1)
```

Comment:
i can't remember if this was gooncode or just copied from other tooltip shit. if you don't want to test it, I'd advise not touching it.